terraform/aws/scenarios/omnibus-tiered-fresh-install/templates/chef-server.rb.tpl

```diff
@@ -14,7 +14,15 @@ server "frontend.internal",
 
 api_fqdn = "frontend.internal"
 
-profiles['root_url'] = 'http://frontend.internal:9998'
+# The public IPV6 address is not bound to the network interface of the machine.
+# rhel-7 and rhel-8 are able to resolve this but the older rhel-6 is not.
+# Using the ipv6 localhost instead of the public IP to fix this.
+require 'ipaddr'
+if IPAddr.new("${front_end_ip}/${cidr}").ipv6?
+  profiles['root_url'] = 'http://[::1]:9998'
+else
+  profiles['root_url'] = 'http://frontend.internal:9998'
```

Comment:
can change this to localhost if we want to be consistent :shrug:

Reply:
I'm fine with localhost, so sorry I went with `frontend.internal`.  I honestly thought it was "more correct" in case someone was running the pedant tests from the backend machine in a tiered scenario.